src/components/CMessageRow.vue

```diff
@@ -24,16 +20,20 @@
     <td class="border px-4 py-2">
       <pre class="text-xs whitespace-normal">{{ waitTime }}</pre>
     </td>
-    <td class="border px-4 py-2">
-      <pre class="text-xs whitespace-normal">{{ progress | percentage }}</pre>
-    </td>
+
     <td class="border px-4 py-2">
       <pre
         v-if="nameState == 'Success' || nameState === 'Failure'"
         class="text-xs  whitespace-normal"
         >{{ executionTime }}</pre
       >
     </td>
+    <td class="border px-4 py-2">
+      <pre class="text-xs whitespace-normal">{{ progress | percentage }}</pre>
+    </td>
+    <td class="border px-4 py-2">
+      <pre class="text-xs whitespace-normal">{{ messageId }}</pre>
```

Comment:
could you put message id in extended row while keeping it accesible via search ?